redwood-widget/build.gradle

```diff
@@ -42,4 +42,18 @@ kotlin {
 
 android {
   namespace 'app.cash.redwood.widget'
+
+  buildFeatures {
+    // Has default ID for RedwoodLayout.
+    androidResources = true
+  }
+}
+
+spotless {
+  kotlin {
+    targetExclude(
+      // Apache 2-licensed files from AOSP.
+      "src/androidMain/kotlin/app/cash/redwood/widget/SavedStateRegistry.kt",
```

Comment:
I'm curious about this one

Reply:
The file started as a copy paste of one from Compose UI which handles adapting SaveableStateRegistry to SavedStateRegistry. I changed the API slightly in the end, but we retain their copyright since it's a majority their code. Unfortunately we don't separate disabling copyright header check from code style check so everything gets disabled.

Reply:
got it, ty!